static/chess.js

```diff
@@ -23,3 +23,33 @@ function buttonFlip(){
     }
     flip = !flip;
 }
+
+function myAjax(){
+let style = getComputedStyle(document.getElementsByClassName("even")[0])
+let style2 = getComputedStyle(document.getElementsByClassName("odd")[0])
+let background = style.backgroundColor
+let background2 =style2.backgroundColor
+color1 = background;
+color2 = background2;
+$.ajax({
+    url : "/flip",
+    type: "POST",
+    data: JSON.stringify({
+        'color1': color1,
+        'color2': color2
+    }),
+    contentType: 'application/json;charset=UTF-8',
+    success: function(response)
+    {
+        console.log(response)
+        for(i = 0; i < table.length/2; i++){
+            document.getElementsByClassName("even")[i].style.backgroundColor = color2;
+            document.getElementsByClassName("odd")[i].style.backgroundColor = color1;
+        }
+    },
+    error: function (response)
+    {
+
+    }
+});
```

Comment:
This looks great! Super nitpicky, but I'd recommend indenting the function body so that it's clear that myAjax() is a function. It just helps readability.